packages/core/components/QuerySidebar/QueryFooter.tsx

```diff
@@ -69,6 +110,53 @@ export default function QueryFooter(props: Props) {
             onClick: props.onQueryDelete,
         },
     ];
+    const saveQueryAsOptions: IContextualMenuItem[] = [
```

Comment:
Might be good to make this a hook or similar to de-duplicate with the list produced in the context menu for saving a manifest

Reply:
moved to hook in [caadd2f](https://github.com/AllenInstitute/biofile-finder/pull/474/commits/caadd2f178cba08d8dc5967369448e1a05ade7aa)